daemon/mgr/container_storage_test.go

```diff
@@ -0,0 +1,56 @@
+package mgr
+
+import (
+	"testing"
+
+	"github.com/alibaba/pouch/apis/types"
+)
+
+func TestSortMountPoint(t *testing.T) {
+	mounts := []*types.MountPoint{
+		{
+			Destination: "/home/admin/log",
+			Source:      "/pouch/volume1",
+		},
+		{
+			Destination: "/var/lib",
+			Source:      "/pouch/volume2",
+		},
+		{
+			Destination: "/home/admin",
+			Source:      "/pouch/volume3",
+		},
+		{
+			Destination: "/home",
+			Source:      "/pouch/volume4",
+		},
+		{
+			Destination: "/opt",
+			Source:      "/pouch/volume5",
+		},
+		{
+			Destination: "/var/log",
+			Source:      "/pouch/volume6",
+		},
+	}
+
+	mounts = sortMountPoint(mounts)
+	if mounts[0].Destination != "/opt" || mounts[0].Source != "/pouch/volume5" {
```

Comment:
I think we can prepare two arrays here. one is out of order, other one is in order. After `sortMountPoint`, use loop to check each item. WDYT?

Reply:
Agree.